### PR TITLE
Modified converter for 3.0.0

### PR DESCRIPTION
**Pull request by skateman**
* Dependencies are now living under the lib folder (like upstream)
* FontAwesome and Bootstrap are included from gems (unlike upstream)
* Dropped dependency on rails-assets-* gems
* Bower dependencies are now production dependencies (like upstream)
* Improved rakefile task dependencies
* NPM packaging not supported yet

### CODE REVIEW

**Comment by skateman** [comment]
@awood @jeff-phillips-18 review please

File: spec/html/layout-alt-fixed-with-footer-inner-scroll.html [hunk 1]
```diff
@@ -25,11 +25,11 @@
     <link rel="apple-touch-icon-precomposed" sizes="57x57" href="../dist/img/apple-touch-icon-precomposed-57.png">
     <link rel="stylesheet" href="../dist/css/patternfly.min.css" >
     <link rel="stylesheet" href="../dist/css/patternfly-additions.min.css" >
-    <script src="../components/jquery/dist/jquery.min.js"></script>
-    <script src="../components/bootstrap/dist/js/bootstrap.min.js"></script>
-    <script src="../components/matchHeight/jquery.matchHeight-min.js"></script>
-    <script src="../components/c3/c3.min.js"></script>
-    <script src="../components/d3/d3.min.js"></script>
+    <script src="//code.jquery.com/jquery-2.1.4.min.js"></script>
```
Comment (awood) [inline spec/html/layout-alt-fixed-with-footer-inner-scroll.html]: Are these links going to resolve?  Looks like a lot of them changed.  If they do resolve, getting rid of the double slash would make it less confusing to the reader.

Comment (skateman) [inline spec/html/layout-alt-fixed-with-footer-inner-scroll.html]: @awood double slash means the same protocol as the current URL's (for https security)

**Comment by awood** [comment]
:+1: from me.

**Comment by jeff-phillips-18** [comment]
LGTM

**Pull request merged**